pkg/actors/internal/reminders/storage/scheduler/scheduler.go

```diff
@@ -124,14 +114,27 @@ func (s *scheduler) CreateReminder(ctx context.Context, reminder *internal.Creat
 		}
 	}
 
+	var failurePolicy *schedulerv1pb.FailurePolicy
+	if reminder.IsOneShot {
```

Comment:
So workflow one shot is in with this PR? if so, I remember seeing a comment from you in one of the files above to delete a reminder that was canceled - should we remove that comment? follow up PR is fine, but wanted to confirm the workflow reminder oneshot bit

Reply:
Yeah that's right- Thanks yeah we should do that. Let's do that in the next PR!